README.md

```diff
@@ -6,11 +6,11 @@ This repository contains the [System.Device.Gpio](https://www.nuget.org/packages
 
 The repository also contains [IoT.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings), a growing set of community-maintained [device bindings](src/devices/README.md) for IoT components.
 
-Note: System.Device.Gpio is in early preview. It is not yet supported and will continue to change. It is currently published to myget (requires use of [nuget.config](samples/led-blink/nuget.config)).
+**Note**: System.Device.Gpio is in early preview. It is not yet supported and will continue to change. It is currently published to myget (requires use of [nuget.config](samples/led-blink/nuget.config)).
```

Comment:
NIT: While you are changing this file, it might be worth changing this note as it is not accurate any longer since preview 2 was released to NuGet so it doesn't need a nuget.Config any longer.

Reply:
This was also noted by @krwq.  Will remove.